### PR TITLE
feat: console submodule

### PR DESCRIPTION
**Pull request by TorstenDittmann**

### CODE REVIEW

File: app/controllers/web/console.php [hunk 1]
```diff
@@ -1,609 +1,19 @@
...
-            ->setParam('header', [$header])
-            ->setParam('footer', [$footer])
-        ;
-
-        $response->html($layout->render());
-    });
-
-App::get('/error/:code')
-    ->groups(['web', 'console'])
+App::get('/console/*')
```
Comment (eldadfux) [inline app/controllers/web/console.php:6]: Does * actually work?!

File: app/controllers/web/console.php [hunk 2]
```diff
@@ -1,609 +1,19 @@
...
+    ->alias('/')
+    ->alias('/invite')
+    ->alias('/login')
+    ->alias('/recover')
+    ->alias('/register')
```
Comment (eldadfux) [inline app/controllers/web/console.php:11]: Very elegant!

File: app/http.php [hunk 3]
```diff
@@ -51,7 +51,7 @@
     Console::success('Reload completed...');
 });
 
-Files::load(__DIR__ . '/../public');
+Files::load(__DIR__ . '/../console');
```
Comment (eldadfux) [inline app/http.php:54]: In theory, I think we can load both folders.

**Review by eldadfux — COMMENTED** [review]
We also need to remove a lot of .phtml files, but not all!

File: app/http.php (hunk 3 above)
Comment (TorstenDittmann) [reply app/http.php:54]: we don't need the public anymore 👍🏻

File: app/controllers/web/console.php (hunk 1 above)
Comment (TorstenDittmann) [reply app/controllers/web/console.php:6]: It looks like 😄

File: Dockerfile [hunk 4]
```diff
@@ -14,12 +14,9 @@ RUN composer install --ignore-platform-reqs --optimize-autoloader \
...
-WORKDIR /usr/local/src/
+COPY app/console /usr/local/src/console
 
-COPY package-lock.json /usr/local/src/
-COPY package.json /usr/local/src/
-COPY gulpfile.js /usr/local/src/
-COPY public /usr/local/src/public
+WORKDIR /usr/local/src/console
 
 RUN npm ci
```
Comment (stnguyen90) [inline Dockerfile:21]: To make better of the image cache, shouldn't we only copy the package.json and package-lock.json first, run `npm ci` and then copy the rest of the files and run `npm run build`?

File: app/controllers/general.php [hunk 5]
```diff
@@ -566,32 +566,6 @@
         );
     });
 
-App::get('/manifest.json')
```
Comment (stnguyen90) [inline app/controllers/general.php:569]: why was this removed?

Comment (TorstenDittmann) [reply app/controllers/general.php:569]: This should be handled by the console, not appwrite 👍🏻 

If at some point we are going to add it for a PWA - we will do it there.

**Review by abnegate — APPROVED** [review]
